server/controllers/list.js

```diff
@@ -875,19 +888,23 @@ const updateItemDetails = (req, resp) => {
         itemToUpdate.link = link;
       }
 
-      if (isOrdered !== null) {
+      if (isOrdered !== undefined) {
         itemToUpdate.isOrdered = isOrdered;
+        updateMessage = `Item "${itemName}" successfully ordered.`;
       }
 
       if (authorId) {
         itemToUpdate.authorId = authorId;
       }
 
+      if (isArchived !== undefined) {
```

Comment:
check if this will be enough `if(!isArchived) `

Reply:
It won't be enough, because sometimes this prop isn't passed to this controller, in solution: `if(!isArchived)` if isArchived or isOrdered prop would be false item wouldn't be updated...